### DISCUSSION TITLE
JSON路径输出的数据错误

### DISCUSSION THREAD

**Discussion by gauson89** [he3-app/he3-feedback#136] [Bug Report]
JSON路径功能通过JSON Path ：$..bicycle 输出的数据是错误的
![image](https://user-images.githubusercontent.com/96934391/232276696-f5be1667-d004-47d5-b149-6d3499d8a117.png)

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues 跟进这个问题：https://github.com/he3-app/he3-feedback/issues/137
